src/main/resources/mcreator.conf

```diff
@@ -1 +1 @@
-mcreator=2021.3
\ No newline at end of file
+mcreator=2021.4
```

Comment:
It's not `2021.4` please change to `2022.1`

Reply:
I wrote 2021.4 before knowing Klemen decided to change it to 2022.1 without saying it to us.

Reply:
But if we take its logic of 2021.1, it was first 2020.6 until 2022, so until 2022, it should be 2021.4 and in January we change it for 2022.1.